src/sql/workbench/contrib/editData/browser/editDataEditor.ts

```diff
@@ -292,11 +293,19 @@ export class EditDataEditor extends BaseEditor {
 			this._setSashDimension();
 
 			this._register(this._sash.onPositionChange(position => this._doLayout()));
+			this._register(this.themeService.onDidColorThemeChange(e => this.updateTheme(e)));
```

Comment:
This is unnecessary - this already extends Themable and so you just need to override this.updateStyles and then call this.getColor to get the color for the current theme.